modules/ROOT/pages/mule-upgrade-tool.adoc

```diff
@@ -427,6 +384,31 @@ The Mule upgrade tool requires both the current Mule instance and the new Mule i
 
 Perform a manual upgrade to ensure that the current and new Mule instances are in version 4.1.5 or later before attempting to upgrade using the Mule upgrade tool.
 
+[[upgrade_clustered_mule_instances]]
+== Upgrading Clustered Mule Instances
+
+The Mule upgrade tool does not automatically support upgrading clustered Mule instances. To upgrade clustered Mule instances, you have to manually upgrade each node using the tool.
```

Comment:
```suggestion
To upgrade clustered Mule instances, use the Mule upgrade tool to automate the upgrade process of your local Mule instances by manually upgrading each node.
```

What do you think about this, Jorge? I find it as a way to link what it says under the "mule upgrade tool" title and what is mentioned  in [Upgrading a Standalone Mule Instance](https://docs.mulesoft.com/release-notes/mule-runtime/updating-mule-4-versions#upgrading-a-standalone-mule-instance).

Reply:
sounds good to me